File: LeetCode_Algorithm/2351(First Letter to Appear Twice)/2351.js
var repeatedCharacter = function(s) {
  // sol1 62ms 79% 42.2MB 32%
  let check = new Set()
  for (let c of s) {
    if (check.has(c)) {
      return c
    }
    check.add(c)
  }  
};

File: LeetCode_Algorithm/2351(First Letter to Appear Twice)/2351.py
'''
LeetCode 2351 : First Letter to Appear Twice
주어진 문자열에서 첫 번째로 두 번 나타나는 글자 구하기

# sol1 31ms 79% 13.8MB 94%
O(n) O(n)
set에 넣어놓고 체크
'''
class Solution:
    def repeatedCharacter(self, s: str) -> str:
        check = set()
        for c in s:
            if c in check:
                return c
            check.add(c)
        